src/crimson/os/seastore/transaction_manager.h

```diff
@@ -108,7 +111,35 @@ class TransactionManager : public ExtentCallbackInterface {
     laddr_t offset) {
     LOG_PREFIX(TransactionManager::get_pin);
     SUBDEBUGT(seastore_tm, "{} ...", t, offset);
-    return lba_manager->get_mapping(t, offset
+    return lba_manager->get_mapping(t, offset, false
+    ).si_then([FNAME, &t](LBAMapping pin) {
+      SUBDEBUGT(seastore_tm, "got {}", t, pin);
+      return pin;
+    });
+  }
+
+  /**
+   * get_containing_pin
+   *
+   * Get the logical pin containing laddr
+   */
+  get_pin_ret get_containing_pin(
+    Transaction &t,
+    laddr_offset_t laddr) {
```

Comment:
After a second thought, let's deal with (encapsulate) alignments only inside the ObjectDataHandler -- e.g. use the aligned type laddr_t at the TM boundary.